packages/vscode-extension/src/service/MessageService.ts

```diff
@@ -1,38 +1,59 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+
 import { window } from 'vscode';
-import { scloudOutput } from '../extension';
+import { Output } from '../extension';
+import { COMMAND, SEE_DETAILS } from './TranslationKeys';
+import { VSTranslationService } from './VSTranslationService';
 
-export class MessageService {
+export default class MessageService {
 	private commandName: string;
+	private translationService: VSTranslationService;
 
 	constructor(commandName: string) {
 		this.commandName = commandName;
+		this.translationService = new VSTranslationService();
 	}
 
-	showTriggeredActionInfo() {
-		window.showInformationMessage(`The ${this.commandName} command has been triggered.`);
+	showTriggeredActionInfo(actionPromise: number, informationMessage: string, statusBarMessage?: string) {
+		window.showInformationMessage(informationMessage);
+		if (statusBarMessage) {
+			window.setStatusBarMessage(statusBarMessage, actionPromise);
+		}
```

Comment:
As the actionPromisie is used for the statusBarMessage we can put the parmaters closer to each other in the signature of this method.
Another question is if we still want to show the pop up (informationMessage) if we are already showing what is happening in the status bar. This is more a question for Carlos but I leave the comment here to not forget it.

Reply:
Something to discuss with the rest of the team, I resolve conversation just for now for closing pull request